client/signup/steps/woocommerce-install/step-store-address/index.tsx

```diff
@@ -52,11 +53,12 @@ export default function StepStoreAddress( props: WooCommerceInstallProps ): Reac
 						onChange={ ( value ) => update( WOOCOMMERCE_STORE_ADDRESS_2, value ) }
 					/>
 
-					<FormCountrySelect
+					<ComboboxControl
```

Comment:
This component does not yet have types in the package definition for `@wordpress/components` which is causing your type errors. I've added them in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58106. Once that merges and the calypso types are updated, this should pass the unit tests. The Calypso team can cause it to pass sooner if you ask!

Reply:
Thanks, we might do that. cc @p-jackson since we're midchat about this :laughing: